### PR TITLE
updated smi specs

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,19 +3,23 @@
 # Generated Code and Docs
 #----------------------------------------------------------------------------------
 
+DEPSGOBIN=$(shell pwd)/_output/.bin
+
 .PHONY: mod-download
 mod-download:
-	go mod download
+	PATH=$(DEPSGOBIN):$$PATH go mod download
```
Comment (ilackarms) [inline Makefile:10]: $$ > $

File: codegen/smi_groups.go [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package codegen
 
 import (
-	"github.com/servicemeshinterface/smi-sdk-go/pkg/apis/split/v1alpha1"
+	access_v1alpha2 "github.com/servicemeshinterface/smi-sdk-go/pkg/apis/access/v1alpha2"
+	specs_v1alpha3 "github.com/servicemeshinterface/smi-sdk-go/pkg/apis/specs/v1alpha3"
+	split_v1alpha1 "github.com/servicemeshinterface/smi-sdk-go/pkg/apis/split/v1alpha1"
+	split_v1alpha3 "github.com/servicemeshinterface/smi-sdk-go/pkg/apis/split/v1alpha3"
```
Comment (ilackarms) [inline codegen/smi_groups.go:7]: i'd like to move away from using underscores in package names: https://blog.golang.org/package-names

File: Makefile (hunk 1 above)
Comment (EItanya) [reply Makefile:10]: https://github.com/solo-io/gloo/blob/1308a6b9b2b73ce857ac7c1a2aaecf8f8a0d9a5c/Makefile#L157
It's makefile weirdness

File: codegen/smi_groups.go (hunk 2 above)
Comment (EItanya) [reply codegen/smi_groups.go:7]: sure

File: Makefile [hunk 3]
```diff
@@ -3,19 +3,23 @@
 # Generated Code and Docs
 #----------------------------------------------------------------------------------
 
+DEPSGOBIN=$(shell pwd)/_output/.bin
```
Comment (ilackarms) [inline Makefile:6]: can we put this inside the `mod-download` target? otherwise it will run every time the makefile is executed

Comment (EItanya) [reply Makefile:6]: This doesn't create it, just define the variable

Comment (ilackarms) [reply Makefile:6]: it will execute pwd every time loading the makefile

**Pull request merged**